[2023-09-28T18:58:52.536+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-09-28T18:58:47.451100+00:00 [queued]>
[2023-09-28T18:58:52.567+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-09-28T18:58:47.451100+00:00 [queued]>
[2023-09-28T18:58:52.569+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-28T18:58:52.570+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-28T18:58:52.573+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-28T18:58:52.637+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-09-28 18:58:47.451100+00:00
[2023-09-28T18:58:52.653+0000] {standard_task_runner.py:55} INFO - Started process 3900 to run task
[2023-09-28T18:58:52.672+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-09-28T18:58:47.451100+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpoe42mnj_']
[2023-09-28T18:58:52.701+0000] {standard_task_runner.py:83} INFO - Job 9: Subtask create_table
[2023-09-28T18:58:52.989+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-09-28T18:58:47.451100+00:00 [running]> on host d9ce0052adca
[2023-09-28T18:58:53.295+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-09-28T18:58:47.451100+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-28T18:58:47.451100+00:00
[2023-09-28T18:58:53.397+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-09-28T18:58:53.424+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
            , parameters: None
[2023-09-28T18:58:53.429+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-09-28T18:58:53.490+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230928T185847, start_date=20230928T185852, end_date=20230928T185853
[2023-09-28T18:58:53.623+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-28T18:58:53.743+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
